Subject: [PATCH] fix sx8 blk driver device naming
X-BK-ChangeSetKey: jgarzik@pobox.com[torvalds]|ChangeSet|20050108214153|41431
Date:	Sat, 08 Jan 2005 23:15:02 +0000
Patch-mainline: yes
From: olh@suse.de


ChangeSet 1.2274, 2005/01/08 13:41:53-08:00, jgarzik@pobox.com

	[PATCH] fix sx8 blk driver device naming
	
	Attached fixes sysfs naming of sx8 block devs to follow LANANA naming.
	
	You then get /sys/block/sx8!0, etc instead of /sys/block/sx80_0 (device
	names should be /dev/sx8/0 instead of /dev/sx80_0)
	
	Signed-off-by: Jeff Garzik <jgarzik@pobox.com>
	Signed-off-by: Andrew Morton <akpm@osdl.org>
	Signed-off-by: Linus Torvalds <torvalds@osdl.org>



 sx8.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)


diff -Nru a/drivers/block/sx8.c b/drivers/block/sx8.c
--- a/drivers/block/sx8.c	2005-01-08 15:09:56 -08:00
+++ b/drivers/block/sx8.c	2005-01-08 15:09:56 -08:00
@@ -1503,7 +1503,7 @@
 		}
 
 		port->disk = disk;
-		sprintf(disk->disk_name, DRV_NAME "%u_%u", host->id, i);
+		sprintf(disk->disk_name, DRV_NAME "/%u", (host->id * CARM_MAX_PORTS) + i);
 		sprintf(disk->devfs_name, DRV_NAME "/%u_%u", host->id, i);
 		disk->major = host->major;
 		disk->first_minor = i * CARM_MINORS_PER_MAJOR;
-
To unsubscribe from this list: send the line "unsubscribe bk-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

